custom_conf.py

```diff
@@ -59,6 +59,29 @@
 # (default is the circle of friends)
 html_favicon = '.sphinx/_static/favicon.png'
 
+
+#############################################################
+# Display the contributor
+import requests
+
+def get_contributors(username, repository):
+    url = f"https://api.github.com/repos/{username}/{repository}/contributors"
+    response = requests.get(url)
+    try:
+        response.raise_for_status()
+        contributors = response.json()
+        contributors_list = [{'name': contributor['login'], 'url': contributor['html_url']} for contributor in contributors]
+        return contributors_list
+    except requests.exceptions.HTTPError as err:
+        print(f"Failed to fetch contributors: {err}")
+        return None
+
+username = 'canonical'
```

Comment:
For most repositories, this is actually not the username, but the organization. To avoid confusion, we should call the variable `github_org` (or something like that).